week_6/src/main/scala/org/epfl/ParallelOrNotParallel.scala

```diff
@@ -0,0 +1,64 @@
+package org.epfl.parallel
+
+import scala.concurrent.{ Await, Future}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.Duration.Inf
+
+/**
+ *  This code compares different ways to kick off two calculations
+ *  in parallel. It also shows a counter example where a for
+ *  comprehension is used in a naive way, which result in the
+ *  calculations running sequentially
+ */
```

Comment:
Maybe move this comment to the `run` method? Or change for `//` so that it is not interpreted as documentation for the method `costlyCalculation1`?